### DISCUSSION TITLE
Support for yggtorrent

### DISCUSSION THREAD

**Discussion by plokijuter** [itsToggle/plex_debrid#543] [Ideas]
Can you provide supports for yggtorrent?
You can search (scrape) freely, then use the ID to download + passkey

https://www3.yggtorrent.wtf/rss/download?id=1081641&passkey=YwDP4q********************QwL****

Tested with Realdebrid and working.

Thx

**Comment by Gamegenie13** [comment]
You can use Jackett (+flaresolverr if needed), it works perfectly.
